classes/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(published_date, on_spotify)
+    super(published_date)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super and @on_spotify == true
+  end
```

Comment:
- [ ] The `and` and `or` keywords are banned. It's just not worth it. Always use `&&` and `||` instead as mentioned in the [Ruby best practices](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md)